src/js/components/Select/index.d.ts

```diff
@@ -32,7 +32,7 @@ export interface BasicSelectProps {
   margin?: MarginType;
   messages?: { multiple?: string };
   name?: string;
-  onChange?: (...args: any[]) => void;
+  onChange?: (event: React.ChangeEvent) => void;
```

Comment:
Same question here. Should this be `React.ChangeEvent<HTMLInputElement>`?
```suggestion
  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
```